common/git-hooks/pre-commit

```diff
@@ -8,7 +8,7 @@
 # and uses the "rush-prettier" autoinstaller.
 
 node common/scripts/install-run-rush.js git-secrets-scan || exit $?
-node common/scripts/install-run-rush.js viperlight-scan || exit $?
+# node common/scripts/install-run-rush.js viperlight-scan || exit $?
```

Comment:
was this not working correctly?

Reply:
viperlight doesnt support windows so it was removed from the precommit hook. It still runs on github as part of the pre-merge workflow